packages/appstore/src/components/wallet-withdrawal/wallet-withdrawal.tsx

```diff
@@ -11,24 +11,19 @@ const WalletWithdrawal = observer(() => {
     const active_wallet = useActiveWallet();
     const { client, modules, ui } = useStore();
     const { is_mobile } = ui;
-    const { iframe } = modules.cashier;
+    const { iframe, withdraw, transaction_history } = modules.cashier;
```

Comment:
@hamid-deriv Would it be possible to not access cashier stores outside of cashier?

Reply:
@farzin-deriv We need to check some conditions from the appstore package.

Reply:
Probably we can check these conditions inside cashier itself 🤔 
@aum-deriv would be working on the withdrawal modules later, we can refactor these in his PR then.